source/tutorial.rst

```diff
@@ -23,36 +23,158 @@ Note that depending on your code style during declaration you may choose to omit
 .. code :: javascript
     
     let x = 4;
-    let y : number = 4;
+    let y: number = 4;
+    let x: boolean = false; 
 
 
-Arithmetic
-^^^^^^^^^^^^
+Arithmetic Expressions
+^^^^^^^^^^^^^^^^^^^^^^
 
 Arithmetic expressions also look a lot like in any other programming language (for good reason!). 
 
-.. code:: javascript
+.. code :: javascript
     
     let x = 4.5;
     x = (x * 2) + 3; // x = 12
-  
+
+
+Boolean Expressions
+^^^^^^^^^^^^^^^^^^^
+Boolean expressions for many are quite familiar, as we have taken inspiration from other programming languages.
+
+**Equality**
+
+We have for equality two binary operators: equals ``==``, not equals ``!=``. 
+
+.. code :: javascript
+
+    let x = 5;
+    let y = x == 5; // y = true
+    let z = x != 6; // z = true
+
+**Comparison**
+
+For comparison we have the following binary operators: less than ``<``, greater than ``>``, less than or equal ``<=`` and greater than or equal ``>=``.
+
+.. code :: javascript
     
+    let x = 5;
+    let y = x < 3;  // y = false
+    let z = x > 4;  // z = true
+    let a = x <= 5; // a = true
+    let b = x >= 6; // b = false
+
+**Logical And and Or**
```

Comment:
Maybe rename this section header to Logical Operators - the "And and" is less readable